pinot-ingestion-jobs/pinot-spark/README.md

```diff
@@ -0,0 +1,93 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+# Pinot Spark
```

Comment:
Please remove this file and add the documentation under readthedocs

Reply:
added Pinot Spark section into readthedocs

Reply:
Can we please remove README.md? It is best if documentation is in one place. 
Thanks

Reply:
Let keep the README. You can point the README to documentation link.

Reply:
yup, that also works as long as contents are in one place. thanks.